### PR TITLE
disbursement bug fixes

### PR DESCRIPTION
**Pull request by tristanhcole**

### CODE REVIEW

File: app/server/api/credit_transfer_api.py [hunk 1]
```diff
@@ -458,7 +458,7 @@ def post(self):
 )
 
 credit_transfer_blueprint.add_url_rule(
-    '/credit_transfer/<int:credit_transfer_id>',
+    '/credit_transfer/<int:credit_transfer_id>/',
```
Comment (tristanhcole) [inline app/server/api/credit_transfer_api.py]: I changed this in [PR 105](https://github.com/teamsempo/SempoBlockchain/pull/105) and don't know why

File: app/server/models/transfer_account.py [hunk 2]
```diff
@@ -198,7 +198,7 @@ def approve_and_disburse(self, initial_disbursement=None, auto_resolve=False):
         from server.utils.access_control import AccessControl
 
         admin = getattr(g, 'user', None)
-        auto_resolve = initial_disbursement != current_app.config['DEFAULT_INITIAL_DISBURSEMENT']
+        auto_resolve = initial_disbursement == current_app.config['DEFAULT_INITIAL_DISBURSEMENT']
```
Comment (tristanhcole) [inline app/server/models/transfer_account.py]: not sure how this wasn't picked up... 🤷‍♂

File: app/server/schemas.py [hunk 3]
```diff
@@ -407,7 +407,7 @@ class TransferUsageSchema(Schema):
 
 transfer_accounts_schema = TransferAccountSchema(
     many=True,
-    only=('balance', 'created', 'id', 'users', 'token', 'primary_user_id', 'blockchain_address')
+    only=('balance', 'created', 'id', 'users', 'token', 'primary_user_id', 'blockchain_address', 'is_approved')
```
Comment (tristanhcole) [inline app/server/schemas.py]: `is_approved` wasn't being returned in transfer account schema but was being accessed on that page...

File: app/server/api/credit_transfer_api.py (hunk 1 above)
Comment (tristanhcole) [reply app/server/api/credit_transfer_api.py]: i think it was to make the tests easier to write...

Comment (tristanhcole) [reply app/server/api/credit_transfer_api.py]: v stupid in hindsight

Comment (enjeyw) [reply app/server/api/credit_transfer_api.py]: I hate how this even matters. Feels like u should be able to just use either.

File: app/server/models/transfer_account.py (hunk 2 above)
Comment (enjeyw) [reply app/server/models/transfer_account.py]: lol!

File: app/server/api/transfer_account_api.py [hunk 4]
```diff
@@ -119,7 +119,7 @@ def put(self, transfer_account_id):
             if not approve == transfer_account.is_approved and transfer_account.is_approved is not True:
                 transfer_account.approve_and_disburse()
 
-            db.session.commit()
+            db.session.flush()
```
Comment (enjeyw) [inline app/server/api/transfer_account_api.py]: i mean i think there's a 90% we'll end up shifting all of these back pretty soon but ok lol

**Pull request merged**